airflow/providers/common/sql/hooks/sql.py

```diff
@@ -37,6 +37,7 @@
 
 import sqlparse
 from deprecated import deprecated
+from more_itertools import chunked
```

Comment:
Do we need to add this as a dependency for the provider?
Upon testing RC Provider [common.sql: 1.11.0rc2](https://pypi.org/project/apache-airflow-providers-common-sql/1.11.0rc2), I am getting the below error

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/airflow/providers/common/sql/operators/sql.py", line 28, in <module>
    from airflow.providers.common.sql.hooks.sql import DbApiHook, fetch_all_handler, return_single_query_results
  File "/usr/local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 40, in <module>
    from more_itertools import chunked
ModuleNotFoundError: No module named 'more_itertools'
```

Reply:
hmm looks like dependency is missing for `more-itertools`

Reply:
https://github.com/apache/airflow/pull/37359